streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -310,8 +310,8 @@ public void shouldConvertValuesIfInnerStoreImplementsTimestampedBytesStore() {
         stateManager.registerStore(
             new WrappedStateStore<NoOpReadOnlyStore<Object, Object>, Object, Object>(store2) {
             },
-            stateRestoreCallback
-        );
+            stateRestoreCallback,
+                null);
```

Comment:
It looks like the formatting got messed up here (and elsewhere). It's a minor issues, but we might as well fix it.